src/main/java/lotto/mainSystem/StartLotto.java

```diff
@@ -0,0 +1,30 @@
+package lotto.mainSystem;
+
+import java.util.List;
+import java.util.Map;
+import lotto.InOutputSystem.InputNums;
+import lotto.InOutputSystem.PrintStrings;
+import lotto.manageLotto.CalcProfit;
+import lotto.manageLotto.CompareLottos;
+import lotto.manageLotto.MakeLottoLists;
+import lotto.manageLotto.MakeWinLotto;
+
+public class StartLotto {
+  public void run(){
+    int inputPrice = InputNums.price();
+
+    MakeLottoLists lottoLists = new MakeLottoLists();
+    List<List<Integer>> myLottos = lottoLists.makeInputLottoList(inputPrice);
+
+    String winningNums = InputNums.winningNums();
```

Comment:
변수명은 줄여쓰지 않는 것이 좋습니다. 조금 길더라도 구체적으로 작성하는 습관을 들이는 게 좋아요
```suggestion
    String winningNumbers = InputNums.winningNums();
```

Reply:
아하 이것이 협업을 위한 디테일